index.esm.js

```diff
@@ -0,0 +1,25 @@
+export * from './controllers';
+export * from './core';
+export * from './elements';
+export * from './platform';
+export * from './plugins';
+export * from './scales';
+
+import * as controllers from './controllers';
+import * as elements from './elements';
+import * as plugins from './plugins';
+import * as scales from './scales';
```

Comment:
what do you mean by `aaa` and `let a = 2` 

 > bbb

ccc
```suggestion
import * as scales from './scales22';
```

Reply:
tt

Reply:
bbcc

Reply:
cc